src/apprentice-giveaway/index.md

```diff
@@ -1,102 +1,55 @@
 ---
-title: Get free access to Flutter Apprentice
-description: You can have free online access to Flutter Apprentice book through January 6, 2022.
+title: Fall Flutter Apprentice event
+toc: false
+description: This event is ended.
 ---
 
-We’re giving you FREE access to Flutter Apprentice from
-October 6, 2021 through January 6, 2022.
+{{site.alert.note}}
+  The "Flutter Apprentice" event is ended.
```

Comment:
Quotation marks, blech! Bold or italics would be better if you think it's really necessary.

```suggestion
  The Flutter Apprentice event has ended.
```